### PR TITLE
Update documentation

### LINKED ISSUE #2368: List all modules in the docs
Not every module is listed into the docs. Make sure to add all available modules.

### PR DESCRIPTION
**Pull request by elyezer**
Include all missing modules and fix formatting warnings.

Closes #2368

### CODE REVIEW

**Comment by Ichimonji10** [comment]
ACK

File: docs/api/robottelo.api.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 
 .. automodule:: robottelo.api
 
+:mod:`robottelo.api.inspect`
+----------------------------
+
+.. automodule:: robottelo.api.inspect
+
 :mod:`robottelo.api.utils`
 ---------------------------
```
Comment (JacobCallahan) [inline docs/api/robottelo.api.rst]: while you're at it, could you remove the addition "-" on this line?

**Comment by JacobCallahan** [comment]
ACK, pending comments

File: docs/api/robottelo.api.rst (hunk 1 above)
Comment (Ichimonji10) [inline docs/api/robottelo.api.rst]: Eagle eyes. :wink:

Comment (JacobCallahan) [inline docs/api/robottelo.api.rst]: OCD

Comment (elyezer) [inline docs/api/robottelo.api.rst]: sure, will update.

**Comment by elyezer** [comment]
@JacobCallahan have updated, thanks for the eagle eyes.

**Comment by JacobCallahan** [comment]
Awesome! Once the tests pass, I'll go ahead and merge

**Pull request merged**